internal/datastore/memdb/readonly.go

```diff
@@ -261,13 +264,24 @@ func (r *memdbReader) mustLock() {
 }
 
 func iteratorForFilter(txn *memdb.Txn, filter datastore.RelationshipsFilter) (memdb.ResultIterator, error) {
-	index := indexNamespace
-	args := []any{filter.ResourceType}
-	if filter.OptionalResourceRelation != "" {
+	index := indexNamespace + "_prefix"
```

Comment:
can you explain why `_prefix` was added to the default index? No index like that can be found in `spicedb/internal/datastore/memdb/schema.go`

Reply:
Its a built-in to the backing memdb for prefix filtering; I'll add a note